docker/ci/build

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -eux
+
+BUNDLER_PATH=.bundle/gems
+
+function build_and_test_image() {
+  container_name="$1"
+
+  : ===
+  : === run linter on the docker files
+  : ===
+   bundle exec puppet-docker local-lint "$container_name"
+
+  : ===
+  : === build and test $container_name
```

Comment:
Hah, I wondered about injections here (suspected it might be ok), and offhand looks like the trivial case at least is ok, i.e. given something like

```sh
container_name='foo;
rm -rf ...
'
```
it's still fully subsumed by the `:` command.